spec/examples.txt

```diff
@@ -1,6 +1,7 @@
 example_id                                 | status | run_time        |
 ------------------------------------------ | ------ | --------------- |
-./spec/features/create_a_user_spec.rb[1:1] | passed | 0.12522 seconds |
-./spec/models/user_spec.rb[1:1]            | passed | 0.00135 seconds |
-./spec/models/user_spec.rb[1:2]            | passed | 0.01491 seconds |
-./spec/models/user_spec.rb[1:3]            | passed | 0.00287 seconds |
+./spec/features/create_a_user_spec.rb[1:1] | passed | 0.11376 seconds |
+./spec/models/user_spec.rb[1:1]            | passed | 0.00403 seconds |
+./spec/models/user_spec.rb[1:2]            | passed | 0.00275 seconds |
+./spec/models/user_spec.rb[1:3]            | passed | 0.00203 seconds |
+./spec/models/user_spec.rb[1:4]            | passed | 0.0116 seconds  |
```

Comment:
I haven't seen this before. Do you know what's generating it  or if you actually want it?